scripts/react-native-xcode.sh

```diff
@@ -107,15 +107,15 @@ fi
 # shellcheck source=/dev/null
 source "$REACT_NATIVE_DIR/scripts/node-binary.sh"
 
-[ -z "$HERMES_PATH" ] && export HERMES_PATH="$REACT_NATIVE_DIR/../hermes-engine-darwin"
+[ -z "$HERMES_PATH" ] && export HERMES_PATH="$REACT_NATIVE_DIR/../hermes-engine"
```

Comment:
Now that we’re switching to the pod version of this, we should load `hermesc` from there instead of this npm package. In the future one might not have this package installed if one doesn’t do Android development.

Reply:
So something like `$PODS_ROOT/hermes-engine/destroot/bin/hermesc` (untested)

Reply:
Yeah, this would be the best solution to go forward.